src/utils/function-panel/getSelectedValue.ts

```diff
@@ -0,0 +1,19 @@
+import { TRequestCondition } from '@pages/filter/ui/panels/function-panel/function-panel.interface'
+
+export const getSelectedValue = (
+  group: string,
+  index: number,
+  requestCondition: TRequestCondition[],
+): string => {
+  let value = '--'
+
+  const currentRequestBlock = requestCondition[index][1]
+
+  Object.entries(currentRequestBlock).map((item: any[]) => {
```

Comment:
It's a not good to use `map` for finding an element in array. Please use `find` method.

Reply:
agree